examples/src/main/scala/example/Common.scala

```diff
@@ -54,8 +56,22 @@ trait Common {
         client.toHttpApp.run(Request[F](Method.GET, uri"http://localhost:8080/client/hello" / name))
       case GET -> Root / "fail" =>
         Concurrent[F].raiseError(new RuntimeException("💥 Boom!"))
-
     }
   }
 
+  def routeClassifier[F[_]]: RouteClassifier = {
+    val dsl = new Http4sDsl[F] {}
+    import dsl._
+
+    RouteClassifier.of[F] {
```

Comment:
It's a bit sad that we need to write it by hand, but it's amazing that we can have classifiers!

Reply:
sadly, I haven't figured out a good way to do it on the client end